configs/crypto-config-ccm-psk-tls1_2.h

```diff
@@ -0,0 +1,30 @@
+/**
+ * \file crypto-config-ccm-psk-tls1_2.h
+ *
+ * \brief Minimal crypto configuration for TLS 1.2 with
+ * PSK and AES-CCM ciphersuites
+ */
+/*
+ *  Copyright The Mbed TLS Contributors
+ *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+ */
+
+/**
+ * To be used in conjunction with configs/config-ccm-psk-tls1_2.h
+ * or configs/config-ccm-psk-dtls1_2.h. */
+
+#ifndef PSA_CRYPTO_CONFIG_H
+#define PSA_CRYPTO_CONFIG_H
+
+#define PSA_WANT_ALG_CCM                        1
+#define PSA_WANT_ALG_CCM_STAR_NO_TAG            1
+#define PSA_WANT_ALG_ECB_NO_PADDING             1
```

Comment:
HMAC is necessary for PSA_WANT_ALG_TLS12_PRF and PSA_WANT_ALG_TLS12_PSK_TO_MS. That's maybe a dependency that we should resolve automatically. Not sure in which file though.

Reply:
> That's maybe a dependency that we should resolve automatically. Not sure in which file though.

Good idea. Also for `PBKDF2_HMAC`, and `PBKDF2_AES_CMAC_PRF_128` with AES + CMAC. This would be in `include/psa/crypto_adjust_config*.h`, none of the existing files fit.